### PR TITLE
add reviews FE+BE connection

### PR DESCRIPTION
**Pull request by maggiechang**
- POST mapping to add review
- PUT mapping to put review in item's reviews list
- FE + BE add reviews connection
- success confirmation page
- click on item takes you to item description/page

### CODE REVIEW

File: front-end/src/components/Success-AddItem.vue [hunk 1]
```diff
@@ -1,10 +1,20 @@
 <template>
     <div class="container message">
             <div class="row justify-content-md-center header-message">
-                <h1><strong>Hooray, you're item has been added to your store.</strong></h1>
```
Comment (mols2ma) [inline front-end/src/components/Success-AddItem.vue:4]: This is embarrassing I’ve truly lost brain cells

Comment (maggiechang) [reply front-end/src/components/Success-AddItem.vue:4]: LOL

**Comment by elishaaquino** [comment]
are there supposed to be back end changes pushed?

**Comment by maggiechang** [comment]
> are there supposed to be back end changes pushed?

Yes Sorry. They're pushed now

**Pull request merged**